### PR TITLE
iOS code snippet for video

### PR DESCRIPTION
**Pull request by ptankTwilio**

### CODE REVIEW

**Comment by aaalaniz** [comment]
+1

File: video/getting-started/use-participant-media/use-participant-media.m [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+participant.delegate = self;
+
+// UIView where you want to render remote participant's video
+@property (nonatomic, weak) IBOutlet UIView *remoteView;
+
+#pragma mark - TVIParticipantDelegate
+
+- (void)participant:(nonnull TVIParticipant *)participant addedVideoTrack:(nonnull TVIVideoTrack *)videoTrack {
+    // Attach remoteView on video track to render video
+    [videoTrack attach:remoteView];
```
Comment (paynerc) [inline video/getting-started/use-participant-media/use-participant-media.m]: Needs to be either `_remoteView` or `self.remoteView` for the compiler to be happy.

File: video/getting-started/use-participant-media/use-participant-media.m [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+- (void)participant:(nonnull TVIParticipant *)participant removedVideoTrack:(nonnull TVIVideoTrack *)videoTrack {
+    // To stop rendering simply call detach
+    [videoTrack detach:remoteView];
```
Comment (paynerc) [inline video/getting-started/use-participant-media/use-participant-media.m]: Needs to be either `_remoteView` or `self.remoteView` for the compiler to be happy.

File: video/getting-started/connect-to-room/connect-to-room.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    builder.localMedia = self.localMedia
+}
+room = client.connect(with: connectOptions, delegate: self)
+
+// MARK: TVIRoomDelegate
+
+func didConnect(to room: TVIRoom) {    
+    // The Local Participant
+    let localParticipant = room.localParticipant;
+    print("Local identity \(localParticipant.identity)")
```
Comment (paynerc) [inline video/getting-started/connect-to-room/connect-to-room.swift]: `localParticipant` is an optional so we'll need to unwrap it.

File: video/getting-started/connect-to-room/connect-to-room.m [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    NSArray *participants = room.participants;
+    NSLog(@"Number of connected participants %ld", [participants count]);
+}
+
+- (void)room:(nonnull TVIRoom *)room participantDidConnect:(nonnull TVIParticipant *)participant {
+    NSLog("Participant %@ has joined Room %@", participant.identity, room.name);
+}
+
+- (void)room:(nonnull TVIRoom *)room participantDidDisconnect:(nonnull TVIParticipant *)participant {
+    NSLog("Participant %@ has left Room %@", participant.identity, room.name);
```
Comment (paynerc) [inline video/getting-started/connect-to-room/connect-to-room.m]: Missing the @ before the log statement.

File: video/getting-started/connect-to-room/connect-to-room.swift [hunk 5]
```diff
@@ -0,0 +1,25 @@
+let connectOptions = TVIConnectOptions { (builder) in
+    builder.name = "my-room"
+    builder.localMedia = self.localMedia
+}
+room = client.connect(with: connectOptions, delegate: self)
```
Comment (paynerc) [inline video/getting-started/connect-to-room/connect-to-room.swift]: Should we change `client` to `videoClient` so we are consistent everywhere?

File: video/getting-started/handle-participants/handle-participants.swift [hunk 6]
```diff
@@ -0,0 +1,25 @@
+let connectOptions = TVIConnectOptions { (builder) in
```
Comment (paynerc) [inline video/getting-started/handle-participants/handle-participants.swift]: Same comments as in `connect-to-room.swift`

File: video/getting-started/setup-local-media/setup-local-media.swift [hunk 7]
```diff
@@ -0,0 +1,18 @@
+// Create a local media object
+var localMedia = TVILocalMedia()
+
+// Add an audio track
+var enable = true;
+var localAudioTrack = localMedia?.addAudioTrack(enable)
```
Comment (paynerc) [inline video/getting-started/setup-local-media/setup-local-media.swift]: `localMedia` is not an optional so it doesn't need unwrapping here.

File: video/getting-started/setup-local-media/setup-local-media.swift [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+// Create camera object
+var camera = TVICameraCapturer()
+
+// Add a video track
+var localVideoTrack = localMedia?.addVideoTrack(enable, capturer: camera!)
```
Comment (paynerc) [inline video/getting-started/setup-local-media/setup-local-media.swift]: `localMedia` is not an optional so it doesn't need unwrapping here.

Comment (paynerc) [inline video/getting-started/setup-local-media/setup-local-media.swift]: `camera` is also not an optional here so force unwrapping it is not necessary here.

File: video/getting-started/setup-local-media/setup-local-media.swift [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+// Remove audio track
+localMedia.removeAudioTrack(localAudioTrack)
```
Comment (paynerc) [inline video/getting-started/setup-local-media/setup-local-media.swift]: `localAudioTrack` is an optional here and requires unwrapping

File: video/getting-started/setup-local-media/setup-local-media.swift [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+// Remove video track
+localMedia.removeVideoTrack(localVideoTrack)
```
Comment (paynerc) [inline video/getting-started/setup-local-media/setup-local-media.swift]: `localVideoTrack` is an optional here and requires unwrapping

**Pull request merged**